pebbles/views/export_stats.py

```diff
@@ -0,0 +1,372 @@
+from flask.ext.restful import reqparse
+from flask import Blueprint as FlaskBlueprint
+from datetime import datetime as dt
+from operator import itemgetter
+
+from pebbles.server import restful, app
+from pebbles.views.commons import auth
+from pebbles.utils import requires_admin
+from pebbles.rules import (apply_rules_export_institutions, apply_rules_export_monthly_instances,
+                           apply_rules_export_quartals)
+
+export_stats = FlaskBlueprint('export_stats', __name__)
+
+date_format = '%Y-%m-%d'
+
+month_count = 12
+
+# institution types
+institution_list = app.config['HAKA_INSTITUTION_LIST']
+
+# Quartal definition
+Q1 = [1, 2, 3]
+Q2 = [4, 5, 6]
+Q3 = [7, 8, 9]
+Q4 = [10, 11, 12]
+
+
+class ExportInstitutions(restful.Resource):
+
+    parser = reqparse.RequestParser()
+    parser.add_argument('start', type=str)
+    parser.add_argument('end', type=str)
+    parser.add_argument('filter', type=str)
+    parser.add_argument('exclude', type=bool)
+
+    @auth.login_required
+    @requires_admin
+    def get(self):
+        args = self.parser.parse_args()
+        if args.get('start') is None and args.get('end') is None:
+            query = apply_rules_export_institutions()
```

Comment:
Validate dates. (Check the start date is lesser than end date).
Example following cases fail:
		1 start-date: 2018-08-30, end-date: 2018-08-29 - exports data
		2 start-date: 2018-08-30, end-date: 2018-07-01 - exports data 
		3 start-date: 2018-08-30, end-date: 2017-07-01 - exports data